docs/workflows/mazer-client.rst

```diff
@@ -0,0 +1,97 @@
+.. _mazer-cli:
+
+Install a Collection, hosted by Pulp, using ``mazer``
+=====================================================
+
+
+Mazer Configuration
+-------------------
+
+`Install mazer <https://galaxy.ansible.com/docs/mazer/install.html#latest-stable-release>`_ and
+use the `url` option to point to the `Distribution` your content should fetch from. For example,
+using the `Distribution` created in the sync workflow, the config would be::
```

Comment:
This paragraph is very vague (ie which url option?)

Reply:
I know :/

Reply:
These docs need a *lot*. I think we need to fully split roles and collections into their own single large parts of the site.

Reply:
That makes sense to me. So let's postpone that.